docs/component-guides/notebooks-guide.md

```diff
@@ -0,0 +1,112 @@
+PYTHONPATH=.. python utils/notebooks/setup_secrets_access.py --region YOUR_CLUSTER_REGION --cluster YOUR_CLUSTER_NAME --profile-namespace YOUR_NAMESPACE
+```  
+
+Use the help flag to learn more about available parameters:
+```bash
+PYTHONPATH=.. python utils/notebooks/setup_secrets_access.py --help
+```
+
+### (Optional) Update default Notebook configurations
+
+No Kubeflow Notebook configuration is selected by default. You can make the `PodDefault` resources that you created the default credential configuration when creating a Notebook. If you do not follow this step, you must manually select this in the Notebook UI. For more information on set up details, see the [Detailed Steps](https://www.kubeflow.org/docs/components/notebooks/quickstart-guide/#detailed-steps) in the Kubeflow Notebooks Quickstart Guide. 
+  
+> Note: Making this configuration default introduces a dependency. The Secrets and PodDefault must be available in all Profile namespaces. If the Secrets and PodDefault resources are not available in a Profile namespaces, newly created Notebook servers in that Profile namespace will fail.
+
+Update the default Kubeflow Notebook configuration either before or after installing Kubeflow. 
+
+#### Option 1: Before installing Kubeflow
+Modify the file `awsconfigs/apps/jupyter-web-app/configs/spawner_ui_config.yaml`
+```yaml
+  configurations:
+    # List of labels to be selected, these are the labels from PodDefaults
+    value:
+      - add-aws-secret
+```  
+#### Option 2: After installing Kubeflow
+Update the Notebook configuration at runtime with the following command:  
+```bash
+kubectl edit $(kubectl get cm -n kubeflow -l app=jupyter-web-app -o=name | grep 'web-app-config') -n kubeflow
+```  
+
+Modify the configuration:  
+```yaml
+  configurations:
+    # List of labels to be selected, these are the labels from PodDefaults
+    value:
+      - add-aws-secret
+```  
+  
+Save and exit your editor. Then, restart the Notebook deployment to apply the changes.   
+
+```shell
+kubectl rollout restart deployment jupyter-web-app-deployment -n kubeflow
+```
+### Verify Notebook credentials
+
+Find `PodDefault` in the Notebook creation page to verify that your setup was done successfully. 
+![](https://user-images.githubusercontent.com/26939775/155630906-0eecf1d9-3fb1-4d01-a85e-1cff46dc37e9.png)  
+
+Create a Notebook and check that the environment variables are accessible.
+```python
+import os
+
+print(os.environ['port'])
+```  
+
+## AWS IAM for Kubeflow Profiles in Notebooks
+
+Use [AWS IAM](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html) to securely access AWS resources through Kubeflow Notebooks.
+
+### Configuration
+
+Prerequisites for setting up AWS IAM for Kubeflow Profiles can be found in the [Profiles component guide](/docs/component-guides/profiles/#configuration-steps). These steps go through creating a profile that uses the `AwsIamForServiceAccount` plugin. No additional configuration steps are required.
```

Comment:
should link to:
https://github.com/judyheflin/kubeflow-manifests-docsy-submodule/blob/kf-aws-site/content/en/docs/component-guides/profiles.md#configuration-steps

`profiles.md#configuration-steps`

Reply:
This does, but the link will only work if you run a local build.